app/src/main/java/com/github/sdp_begreen/begreen/firebase/FirebaseDB.kt

```diff
@@ -126,6 +126,12 @@ object FirebaseDB: DB {
         }
     }
 
+    override suspend fun findUserByName(name: String, timeout: Long): List<User> {
+        return getAllUsers(timeout).filter {
+            it.displayName?.contains(name) ?: false
+        }
+    }
+

```

Comment:
I am not sure to see why you added this method, as it is not used in the project for now ?

Do you plan on using it later ? Otherwise, I think maybe we should remove it if we don't use it.

Reply:
Nevermind, thats an artifact from my first task, I thought I'd need that function later on but it turned out I didnt. 
You're right, i'll delete it, thanks!